src/app/components/Operator/components/ValidatorsInOperatorTable.tsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import { Box } from '@material-ui/core';
 import Link from '@material-ui/core/Link';
-import Typography from '@material-ui/core/Typography';
 import config from '~app/common/config';
 import { useStyles } from '~app/components/Styles';
 import { longStringShorten } from '~lib/utils/strings';
+import Typography from '@material-ui/core/Typography';
 import BeaconchaLink from '~app/common/components/BeaconchaLink';
 import DataTable from '~app/common/components/DataTable/DataTable';
 import CopyToClipboardIcon from '~app/common/components/CopyToClipboardIcon';
+import IsValid from '~app/common/components/IsValid/IsValid';
+import Status from '~app/common/components/Status';
```

Comment:
Sort all the imports from shorter to longer.
First node modules sorted, then app components sorted, import React always goes first no matter what.

Reply:
1. Sorting is not done
2. Import should be: `import IsValidBadge from '~app/common/components/IsValidBadge';`